.eslintrc.js

```diff
@@ -23,6 +23,36 @@ module.exports = {
      */
     'no-param-reassign': ['error', { props: false }],
 
+    /**Disallows unnecessary return await
+     * https://eslint.org/docs/rules/no-return-await
+     */
+    'no-return-await': ['error'],
+
+    /**
+     * Disallow using an async function as a Promise executor
+     * https://eslint.org/docs/rules/no-async-promise-executor
+     */
+    'no-async-promise-executor': ['error'],
+
+    /**
+     * Disallow await inside of loops
+     * https://eslint.org/docs/rules/no-await-in-loop
+     */
+    'no-await-in-loop': ['error'],
+
+    /**
+     * Disallow assignments that can lead to race conditions due to
+     * usage of await or yield
+     * https://eslint.org/docs/rules/require-atomic-updates
+     */
+    'require-atomic-updates': ['error'],
+
+    /**
+     * Disallow async functions which have no await expression
+     * https://eslint.org/docs/rules/require-await
+     */
+    'require-awaut': ['error'],
```

Comment:
```suggestion
    'require-await': ['error'],
```

Reply:
Also, this one seems to create a conflict with the rule in line 29 `no-return-await` in cases like [line 71 in storage.js](https://github.com/Seneca-CDOT/telescope/blob/cda82ab740464c04d50605bd6576382809381311/src/backend/utils/storage.js#L71).

Reply:
I think we can probably fix line 71 with this

```
getPost: async guid => {
  const posts = await redis.getall(guid);
  return posts;
  },
```